contracts/EscrowedMultipleMerkleDistributor.sol

```diff
@@ -98,10 +98,11 @@ contract EscrowedMultipleMerkleDistributor is
         // mark it claimed and send the token to RewardEscrow
         _setClaimed(index, epoch);
         IERC20(token).approve(rewardEscrow, amount);
-        IRewardEscrow(rewardEscrow).createEscrowEntry(
+        IRewardEscrowV2(rewardEscrow).createEscrowEntry(
             account,
             amount,
-            52 weeks
+            52 weeks,
+            90
```

Comment:
I know the fewer changes, the better, but there are a lot of raw numbers being used, like 52 weeks and 90, that would be better as constants.

Reply:
Did you one better, grabbed the constants directly from RewardEscrowV2

Reply:
so clean ✨